src/consensus/pbft_manager.cpp

```diff
@@ -1672,6 +1673,8 @@ bool PbftManager::pushPbftBlock_(SyncBlock &sync_block, vec_blk_t &dag_blocks_or
   // Commit DB
   db_->commitWriteBatch(batch);
```

Comment:
I would just add the mutex here and but splitting this function is not good as we can forgot to remove transaction in the future